m4_sessions/1706-inning/group_four.md

```diff
@@ -2,8 +2,62 @@
 
 ### Session Title:
 
+"Getting involved and giving back as a beginner"  
```

Comment:
Great start.  Be sure to build out timing for each section and have your tools well built out.  Divide and conquer the work next planning session.